packages/syft-libs/pyscaffoldext-syft-support/src/pyscaffoldext/syft_support/script.py

```diff
@@ -318,19 +306,16 @@ def main() -> None:
     package_support["lib"] = package_name
     # petlib doesnot have version
     # package_support["Version"] = package.__version__
-    package_support["class"] = list(classes_set)
+     # sort it all
+    classlist = list(classes_set)
+    classlist.sort()
```

Comment:
Q: Any specific reason who you are sorting?

Reply:
Madhava requested this for convenience while reading the json